src/components/rewind/rewind.js

```diff
@@ -4,7 +4,8 @@ import {h} from 'preact';
 import {Localizer, Text} from 'preact-i18n';
```

Comment:
@dan-ziv  - even though this isn't a doc, should I perhaps be reviewing the tooltips?